### DISCUSSION TITLE
User Experience Performance Measurements are possible/realistics?

### DISCUSSION THREAD

**Discussion by fmancardi** [getgauge/taiko#1988] [Development]
I've received the following assignment: we need to measure the application performance that users 'feel/see/suffers' on the browser.  
It's possible to use taiko automation scripts and the taiko-diagnostics plugin to get execution times, that have a value? i.e. that are not 'heavily' altered by taiko itself?

any tip/hint is welcomed

**Comment by sriv** [comment]
@fmancardi not sure I understand the "not 'heavily' altered by taiko itself" bit - but if you are worried if taiko/diagnostics would add any overhead, then in theory - yes, but I'd be surprised if it is significant in the context. taiko-diagnostics works similar to how lighthouse would.
